engine/core/src/test/java/org/datacleaner/configuration/DataCleanerConfigurationUpdaterTest.java

```diff
@@ -29,13 +29,12 @@
 import org.datacleaner.util.SecurityUtils;
 import org.junit.Assert;
 import org.junit.Test;
+import org.w3c.dom.NodeList;
 
 public class DataCleanerConfigurationUpdaterTest {
 
-    private String originalPassword = encode("admin");
-    private String newPassword = encode("newPassword");
     private String tagName = "password";
-    private String nodePathString = "descriptor-providers:remote-components:server:" + tagName;
+    private String nodePathString = "configuration/descriptor-providers/remote-components/server/" + tagName;
 
     private final Resource originalConfigurationResource = new FileResource("src/test/resources/conf.xml");
```

Comment:
I think the filename here (conf.xml) should be named so that it is recognizable which test is using it, or else I look at src/test/resources and think it is just a general conf.xml file.